angular/Diagram/Gridlines.md

```diff
@@ -123,7 +123,7 @@ export class ModelComponent {
 
 ![Line Intervals in Angular Diagram](/angular/Diagram/Gridlines_images/Gridlines_img2.png)
 
-# Snapping
+## Snapping
```

Comment:
Below Hierarchy is not proper. Please check.

Reply:
done